memberlist.go

```diff
@@ -495,8 +495,18 @@ func (m *Memberlist) SendTo(to net.Addr, msg []byte) error {
 	buf[0] = byte(userMsg)
 	buf = append(buf, msg...)
 
+	// Lookup node from address
+	toAddr := strings.Split(to.String(), ":")[0]
```

Comment:
This'll crash if the split doesn't work - let's use https://golang.org/pkg/net/#SplitHostPort and check the error return from that.